lib/go/thrift/ssl_socket.go

```diff
@@ -20,152 +20,155 @@
 package thrift
```

Comment:
Sorry, I'm using the geany editor and I have it set to use 2 character spacing to match the rest of the codebase (C++ mostly).  I'll undo the whitespace changes here.